configuration/driver/icedrv_forcing.F90

```diff
@@ -454,6 +455,8 @@ subroutine get_forcing(timestep)
 
         opening(:) =   c1intp * open_data(mlast) + c2intp * open_data(mnext)
         closing(:) = -(c1intp * clos_data(mlast) + c2intp * clos_data(mnext))
+        !opening(:) = open_data(timestep)
+        !closing(:) = -1*clos_data(timestep)
```

Comment:
If these lines are just for your local testing, please remove them.

Reply:
done.